endpoints/openrtb2/amp_auction.go

```diff
@@ -143,16 +143,16 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	defer cancel()
 
 	usersyncs := usersync.ParsePBSCookieFromRequest(r, &(deps.cfg.HostCookie))
-	if req.App != nil {
-		labels.Source = pbsmetrics.DemandApp
+	labels.Source = pbsmetrics.DemandWeb
```

Comment:
Why is the `Source` label changing from `DemandApp` to `DemandWeb`? Shouldn't it be `DemandApp` as it's an amp request?

Reply:
in amp_auction we don't expect to ever have an App field so the entire `req.App` conditional can be discarded because we won't be using that value at all

Reply:
I don't think it hurts to check that but the question is shouldn't `labels.Source` be assigned `pbsmetrics.DemandApp`? Why is that changing to `pbsmetrics.DemandWeb` ?

Reply:
As my understanding, it should be a Site.

Reply:
so we should use `DemmandWeb`